tests/progaudi.tarantool.tests/Schema/GetSpace_Should.cs

```diff
@@ -81,5 +82,17 @@ public class GetSpace_Should : TestBase
                 }
             }
         }
+
+        [Fact]
+        public async Task iterating_over_schema()
+        {
+            using (var tarantoolClient = await Client.Box.Connect(ConnectionStringFactory.GetReplicationSource_1_7()))
+            {
+                foreach (var pair in (IReadOnlyDictionary<string, ISpace>)tarantoolClient.Schema)
```

Comment:
This will be pain in the ass, because you always forced to cast schema to exact interface.